### ISSUE TITLE
editing jsonschema itself

### ISSUE THREAD

**Issue by trebor** [josdejong/jsoneditor#1343]
i've been going round and round trying to use jsoneditor to edit jsonschema itself.  i can't find any documentation or examples of this use case after much searching in the [API documentation](https://github.com/josdejong/jsoneditor/blob/master/docs/api.md) and in [Ajv](ajv.js.org).  i am happy to detail my journey and create a working example to illustrate the problems i have encountered.  that said i suspect that this is a most obvious use case and that there is an easy answer i have simply missed.  can anyone point me in the right direction?

**Comment by josdejong** [comment]
You can open the JSON Schema document as a regular JSON document in the editor.

**Comment by trebor** [comment]
yes, that is exactly where i started, and i think possibly the most obvious thing to try.  how would you specify the schema to validate that document?

**Comment by josdejong** [comment]
So you mean you're looking for the JSON schema to validate a JSON schema document?

I think ajv has what you're looking for: https://github.com/ajv-validator/ajv/tree/master/lib/refs
